src/main/java/com/solvd/micro9/users/web/controller/exception/ExceptionResolver.java

```diff
@@ -0,0 +1,25 @@
+package com.solvd.micro9.users.web.controller.exception;
+
+import graphql.GraphQLError;
+import graphql.GraphqlErrorBuilder;
+import graphql.schema.DataFetchingEnvironment;
+import org.springframework.graphql.execution.DataFetcherExceptionResolverAdapter;
+import org.springframework.graphql.execution.ErrorType;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ExceptionResolver extends DataFetcherExceptionResolverAdapter {
```

Comment:
@eyakauleva nice solution

since GraphQL always throws `200` HTTP status code it's very inconvenient to show errors to the end user

Reply:
@h1alexbel GraphQL still returns 200 HTTP status code. This resolver sets status code only as a field to a response body and also sets custom error message there. 
As I found out, it's a GraphQL standard to always return 200 OK in such cases, cause non-200 response codes indicate that some issue occurred at the HTTP transport layer, not the GraphQL layer.
So I'm not sure whether it's a good practice to replace GraphQL HTTP status code and whether it's possible at all.